src/components/SaleCard/SaleCard.tsx

```diff
@@ -0,0 +1,180 @@
+}
+
+const SaleCard = forwardRef<HTMLDivElement, SaleCardProps>(
+  (
+    {
+      className,
+      imageSrc,
+      imageAlt = 'Auction Image',
+      auctionType,
+      titleText,
+      date,
+      location,
+      primaryButtonText,
+      primaryButtonOnClick,
+      secondaryButtonText,
+      secondaryButtonOnClick,
+      badgeText,
+      modalButtonOnClick,
+      modalButtonText,
+      pdfLinkText,
+      pdfLinkUrl,
+      ...props
+    },
+    ref,
+  ) => {
+    const { className: baseClassName, ...commonProps } = getCommonProps(props, 'SaleCard');
+    const componentProps = { ...commonProps, ...props };
+
+    return (
+      <div {...componentProps} className={classnames(baseClassName, className)} ref={ref}>
+        {imageSrc ? (
+          <SeldonImage aspectRatio="16/9" src={imageSrc} alt={imageAlt} className={`${baseClassName}__image`} />
+        ) : null}
+        <div className={`${baseClassName}__details`}>
+          <Text variant={TextVariants.badge}>{auctionType}</Text>
+          <Text variant={TextVariants.title4}>{titleText}</Text>
+          {badgeText && (
+            <Text variant={TextVariants.badge} className={`${baseClassName}__badge`}>
+              {badgeText}
+            </Text>
+          )}
+          <div className={`${baseClassName}__info`}>
+            <Text variant={TextVariants.string2}>{location}</Text>
+            <Text variant={TextVariants.string2}>{date}</Text>
+            {modalButtonText && modalButtonOnClick && (
+              <div className={`${baseClassName}__modal-link`}>
+                <Button
+                  onClick={modalButtonOnClick}
+                  variant={ButtonVariants.tertiary}
+                  className={`${baseClassName}__modal-link-button`}
+                >
+                  {modalButtonText}
+                </Button>
+              </div>
+            )}
+          </div>
+        </div>
+        <SSRMediaQuery.Media greaterThanOrEqual="md">
+          <div className={`${baseClassName}__cta`}>
+            {primaryButtonText && primaryButtonOnClick && (
+              <Button
+                variant={ButtonVariants.primary}
+                onClick={primaryButtonOnClick}
+                className={`${baseClassName}__cta-button`}
+              >
+                {primaryButtonText}
+              </Button>
+            )}
+            {secondaryButtonText && secondaryButtonOnClick && (
+              <Button
+                variant={ButtonVariants.secondary}
+                onClick={secondaryButtonOnClick}
+                className={`${baseClassName}__cta-button`}
+              >
+                {secondaryButtonText}
+              </Button>
+            )}
+            {pdfLinkText && pdfLinkUrl && (
+              <div className={`${baseClassName}__pdf-link`}>
+                <Button href={pdfLinkUrl} target="_blank" variant={ButtonVariants.tertiary}>
+                  {pdfLinkText}
+                </Button>
+              </div>
```

Comment:
I believe the design still show this pdf download button as a secondary. We can leverage the secondary button and just conditionally pass either a `pdfLinkURL` or `secondaryButtonOnClick`. And if you need styles conditionally rennder an additional pdf link class based of the url props existence.